.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Jekyll",
+  "image": "mcr.microsoft.com/devcontainers/jekyll:2-bullseye",
+  "onCreateCommand": "git config --global --add safe.directory ${containerWorkspaceFolder}",
+  "postCreateCommand": "bash .devcontainer/post-create.sh",
+  "customizations": {
+    "vscode": {
+      "settings": {
+        "terminal.integrated.defaultProfile.linux": "zsh",
+        // Speed up extension installation
+        "extensions.verifySignature": false
```

Comment:
Do we have a strong reason for that? Just curious if it's a good idea.

Reply:
May I ask which line of code you are referring to?

Reply:
Line 11:

```json
        // Speed up extension installation
        "extensions.verifySignature": false
```

Reply:
It is necessary, in my experience, almost every time, the installation of the extensions takes too long, although the devcontainer has been started, but the code formatting extension is not available,  the status bar at the bottom of the vscode keeps showing "Configuring Dev Container ...".

Reply:
I see. Thanks for sharing.

Reply:
JFYI: There was an [issue](https://github.com/microsoft/vscode-remote-release/issues/9628) related to that but it seems to be fixed now.

Reply:
I've also read the issue, and the feedback from the dev team is that it's currently fixed in the vscode `insiders` edition but not yet released.

https://github.com/microsoft/vscode-remote-release/issues/9628#issuecomment-2113295839